internal/pkg/manifest/job.go

```diff
@@ -135,6 +135,11 @@ func (j ScheduledJob) ApplyEnv(envName string) (WorkloadManifest, error) {
 	return &j, nil
 }
 
+// TaskPlatform returns the os/arch for the service. This is an empty string if the default (linux/amd64) is detected.
+func (s *ScheduledJob) TaskPlatform() string {
```

Comment:
Perhaps we can take advantage of go's composition and define a function with `TaskConfig` being the receiver:
```suggestion
func (t *TaskConfig) TaskPlatform() string {
  return t.Platform
}
```
This way we don't need to maintain several functions for a single purpose - same with `AppRunnerInstanceConfig`!

Reply:
So good! Thanks!